toxics/limit_data.go

```diff
@@ -0,0 +1,47 @@
+package toxics
+
+import "github.com/Shopify/toxiproxy/stream"
+
+// LimitDataToxic has limit in bytes
+type LimitDataToxic struct {
+	Bytes int64 `json:"bytes"`
+}
+
+func (t *LimitDataToxic) Pipe(stub *ToxicStub) {
+	var bytesRemaining = t.Bytes
```

Comment:
Unfortunately the way toxics are loaded and interrupted, this value won't be persistent.
There's several cases were a toxic is interrupted and the `Pipe()` function is restarted.

I think the solution to this is to put a private `bytesTransmitted` variable in the toxic struct. It will default to 0 when the toxic is created, and you can increment it when you send bytes.

I'll have to check through the rest of the toxiproxy code to make sure there's no places were this value would get reset.